apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/EvaluationStatusBanner/index.tsx

```diff
@@ -1,82 +1,31 @@
 'use client'
 
-import { useCallback, useEffect, useRef, useState } from 'react'
+import { ProgressIndicator } from '@latitude-data/web-ui'
+import { isEvaluationRunDone } from '$/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_lib/isEvaluationRunDone'
+import { type EventArgs } from '$/components/Providers/WebsocketsProvider/useSockets'
 
-import { EvaluationDto } from '@latitude-data/core/browser'
-import { ProgressIndicator, useCurrentDocument } from '@latitude-data/web-ui'
-import {
-  useSockets,
-  type EventArgs,
-} from '$/components/Providers/WebsocketsProvider/useSockets'
-
-const DISAPERING_IN_MS = 5000
 export function EvaluationStatusBanner({
-  evaluation,
+  jobs,
 }: {
-  evaluation: EvaluationDto
+  jobs: EventArgs<'evaluationStatus'>[]
 }) {
-  const timeoutRef = useRef<number | null>(null)
-  const [jobs, setJobs] = useState<EventArgs<'evaluationStatus'>[]>([])
-  const document = useCurrentDocument()
-
-  const onMessage = useCallback(
-    (args: EventArgs<'evaluationStatus'>) => {
-      if (evaluation.id !== args.evaluationId) return
-      if (document.documentUuid !== args.documentUuid) return
-
-      setJobs((prevJobs) => {
-        const jobIndex = prevJobs.findIndex(
-          (job) => job.batchId === args.batchId,
-        )
-
-        if (jobIndex === -1) {
-          return [...prevJobs, args]
-        } else {
-          const newJobs = [...prevJobs]
-          newJobs[jobIndex] = args
-
-          if (isDone(args)) {
-            setTimeout(() => {
-              setJobs((currentJobs) =>
-                currentJobs.filter((job) => job.batchId !== args.batchId),
-              )
-            }, DISAPERING_IN_MS)
-          }
-
-          return newJobs
-        }
-      })
-    },
-    [evaluation.id, document.documentUuid],
-  )
-
-  useEffect(() => {
-    return () => {
-      if (timeoutRef.current) {
-        clearTimeout(timeoutRef.current)
-      }
-    }
-  }, [])
-
-  useSockets({ event: 'evaluationStatus', onMessage })
-
   return (
     <>
       {jobs.map((job) => (
         <div key={job.batchId} className='flex flex-col gap-4'>
-          {!isDone(job) && (
+          {!isEvaluationRunDone(job) && (
             <ProgressIndicator state='running'>
               {`Running batch evaluation ${job.completed}/${job.total}`}
             </ProgressIndicator>
           )}
-          {job.errors > 0 && !isDone(job) && (
+          {job.errors > 0 && !isEvaluationRunDone(job) && (
```

Comment:
actually better to not remove this message even when it's done

Reply:
How they can remove the message?

Reply:
navigating away